src/App.js

```diff
@@ -1,28 +1,33 @@
-import './App.css';
-import Header from './components/Header';
-import LoggedIn from './components/LoggedIn';
-import WelcomeParagraph from './components/WelcomeParagraph';
-import Nav from './components/Nav';
+import "./App.css";
+import Header from "./components/Header";
+import LoggedIn from "./components/LoggedIn";
+import WelcomeParagraph from "./components/WelcomeParagraph";
+import Nav from "./components/Nav";
 import { Routes, Route } from "react-router-dom";
-import Reviews from './components/Reviews';
-import IndividualReview from './components/IndividualReview';
-import ReviewComments from './components/ReviewComments';
+import Reviews from "./components/Reviews";
+import IndividualReview from "./components/IndividualReview";
+import ReviewComments from "./components/ReviewComments";
 
 function App() {
-  const LoggedInUser = "BakriKhalifa"
+  const LoggedInUser = "BakriKhalifa";
```

Comment:
You are currently posting comments as "weegembump" - this is now a little misleading - consider changing this `LoggedInUser` value to "weegembump" for the time being